Firmware/Marlin_main.cpp

```diff
@@ -10939,46 +10927,37 @@ void restore_print_from_eeprom(bool mbl_was_active) {
     float pos_y = eeprom_read_float((float*)(EEPROM_UVLO_CURRENT_POSITION + 4));
     if (pos_x != X_COORD_INVALID)
     {
-        sprintf_P(cmd, PSTR("G1 X%f Y%f F3000"), pos_x, pos_y);
-        enquecommand(cmd);
+        enquecommandf(PSTR("G1 X%f Y%f F3000"), pos_x, pos_y);
```

Comment:
This format string should be clamped to some fixed size ... a single %f could theoretically produce up to 19 characters ... and having 2 of them in the format string will overflow the 30 bytes buffer in `enquecommandf`.